scripts/ci/proxy/deploy.sh

```diff
@@ -18,7 +18,7 @@ kubectl apply -f "${dir}/nginx-proxy-plain-http.yaml"
 cert_dir="${PROXY_CERTS_DIR:-$(mktemp -d)}"
 
 echo "Cert directory: ${cert_dir}"
-"${dir}/../../../tests/scripts/setup-certs.sh" "${cert_dir}" "central-proxy.stackrox.local" "Proxy CA"
```

Comment:
The test coverage is not affected by using `localhost` as long as Sensor does not connect to central via the proxy.
This is not the case in our e2e-tests. Our proxy tests are tested only inside the cluster and don't rely on DNS names set from the outside.